docs/topics/http/sessions.txt

```diff
@@ -701,9 +700,9 @@ In order to build a custom session engine or to customize an existing one, you
 may create a new class inheriting from :class:`~backends.base.SessionBase` or
 any other existing ``SessionStore`` class.
 
-Extending most of the session engines is quite straightforward, but doing so
-with database-backed session engines generally requires some extra effort (see
-the next section for details).
+You can generally extend the session engines, but doing so with database-backed
+session engines generally requires some extra effort (see the next section for
+details).
```

Comment:
Double "generally" doesn't read well. I think dropping the first one still reads fine: "You can extend the session engines, but doing so..."